util/logger/config.go

```diff
@@ -1,5 +1,10 @@
 package logger
 
+const (
+	LogDirectory = "logs"
+	LogFilename  = "pactus.log"
```

Comment:
Please move this from `config.go` to `logger.go`
It is not something to be configured.